lighthouse-core/report/html/renderer/standalone.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global document window ga */
+
+import {DOM} from './common/dom.js';
+import {Logger} from './common/logger.js';
+import {ReportRenderer} from './common/report-renderer.js';
+import {ReportUIFeatures} from './common/report-ui-features.js';
+
+function __initLighthouseReport__() {
+  const dom = new DOM(document);
+  const renderer = new ReportRenderer(dom);
+  const container = dom.find('main', document);
+  /** @type {LH.ReportResult} */
+  // @ts-expect-error
+  const lhr = window.__LIGHTHOUSE_JSON__;
+  renderer.renderReport(lhr, container);
+
+  // Hook in JS features and page-level event listeners after the report
+  // is in the document.
+  const features = new ReportUIFeatures(dom);
+  features.initFeatures(lhr);
+}
+
+if (document.readyState === 'loading') {
+  window.addEventListener('DOMContentLoaded', __initLighthouseReport__);
+} else {
```

Comment:
this seems new, isn't `__LIGHTHOUSE_JSON__` always going to be undefined when this script executes? now that we moved it into the `%%LIGHTHOUSE_JAVASCRIPT%%` block?